docs/modules/deployment/pages/serverless-faas.adoc

```diff
@@ -8,3 +8,222 @@ include::ROOT:partial$version-check.adoc[]
 == AWS Lambda
 
 You can deploy Cerbos to AWS Lambda by building a special container image that includes the Lambda runtime and the Cerbos binary. See https://github.com/cerbos/cerbos-aws-lambda for an example. The repository also contains an example of an AWS Lambda function that creates an AWS API Gateway endpoint to communicate with Cerbos over the HTTP protocol.
+
+== Fly.io
+
+You can deploy Cerbos with link:https://fly.io/docs/apps[Fly Launch] by installing
+link:https://fly.io/docs/hands-on/install-flyctl[flyctl], creating a `fly.toml` configuration file and executing the
+command `flyctl launch` at the directory `fly.toml` file rests.
+
+An example `fly.toml` configuration to deploy Cerbos which gives access to the Cerbos default HTTP port 3592 and the
+gRPC port 3593, and enables the healthchecks and metrics:
+[source,toml,linenums]
+----
+app = '<APPLICATION_NAME>' <1>
+primary_region = 'lhr'
```

Comment:
```suggestion
primary_region = '<REGION>' <2>
```

The users should change this as well by choosing a region from https://fly.io/docs/reference/regions/#fly-io-regions